pkg/virt-controller/watch/vmi/vmi.go

```diff
@@ -178,6 +179,8 @@ type annotationsGenerator interface {
 
 type statusUpdater func(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod) error
 
+type validateSpec func(*k8sfield.Path, *virtv1.VirtualMachineInstanceSpec, *virtconfig.ClusterConfig) []v1.StatusCause
```

Comment:
Lets keep it similar to what we've tried to do in the VMI creation admitter.
```suggestion
type specValidator func(*k8sfield.Path, *virtv1.VirtualMachineInstanceSpec, *virtconfig.ClusterConfig) []v1.StatusCause
```

Reply:
Done